src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -4217,6 +4217,27 @@ internal Token GetMemberAccessOperator(bool allowLBracket)
                 return NewToken(TokenKind.LBracket);
             }
 
+
+            if (ExperimentalFeature.IsEnabled("PSNullConditionalOperators") && c == '?')
+            {
+                _tokenStart = _currentIndex;
+                SkipChar();
```

Comment:
Will this affect `abc.def??2`?
The first `?` is skipped in this case.

Reply:
Do you mean `${abc}.def??2`? In this case the it is parsed for `??` operator.

Reply:
You are calling `SkipChar` here, so the first `?` is gone, my concern is if it will affect `??` and `? :` in any cases.

Reply:
We do an `UngetChar()` if it is not `?.` or `?[`.